decidim-core/config/brakeman.ignore

```diff
@@ -1,5 +1,35 @@
 {
   "ignored_warnings": [
+    {
+      "warning_type": "Cross-Site Scripting",
+      "warning_code": 2,
+      "fingerprint": "211ff4b5e0d738e40e3c7f6d27b6905f23b1ed4e20347c179af3df40f6e5694d",
+      "check_name": "CrossSiteScripting",
+      "message": "Unescaped model attribute",
+      "file": "app/views/decidim/messaging/conversations/_conversation.html.erb",
+      "line": 20,
+      "link": "https://brakemanscanner.org/docs/warning_types/cross_site_scripting",
+      "code": "username_list((Unresolved Model).new.interlocutors(current_user), :shorten => true)",
```

Comment:
I don't like adding a potential XSS but... 
1. I couldn't find a way of saving a name/nickname with an XSS (I get an {:error=>:invalid})
2. At least with brakeman we know where's the potential problem and we can tackle it in the future